devsetup/tripleo/ceph.sh

```diff
@@ -38,14 +38,6 @@ data_devices:
     - /dev/ceph_vg0/ceph_lv0
 EOF
 
```

Comment:
so, the adoption test suite worked with that change, however the test migration has failed because of health warning. I don't think that change could become the cause?
```
* periodic-adoption-multinode-to-crc-no-ceph: # should work w/o ext net, and tripleo netconfig re-applied
	- overcloud deploy failed https://logserver.rdoproject.org/99/54199/12/check/periodic-adoption-multinode-to-crc-no-ceph/be8aa7b/controller/ci-framework-data/logs/ci_script_010_run_tripleo.log
		so, the external network must be attached for tripleo in CI
			2024-10-29 15:35:28.462877 | fa163e0d-e345-2420-70e2-00000000a221 |      FATAL | Ensure certificate requests | np0004972888
* periodic-adoption-multinode-to-crc-ceph: # should work when attach_external_net set, and tripleo netconfig re-applied
	- overcloud deployed https://logserver.rdoproject.org/99/54199/12/check/periodic-adoption-multinode-to-crc-ceph/78de32a/controller/ci-framework-data/logs/ci_script_010_run_tripleo.log
		so, we can safely reapply tripleo netconfig in CI idempotently:
			2024-10-29 11:41:52.179603 | fa163e24-0dfc-1b00-9b33-00000000005e |       TASK | Run NetworkConfig with tripleo_os_net_config
			2024-10-29 11:41:52.248986 | fa163e24-0dfc-1b00-9b33-00000000005e |     TIMING | tripleo_network_config : Run NetworkConfig with tripleo_os_net_config | undercloud | 0:00:35.350506 | 0.07s
	- ceph migrate failed, and the job failed https://logserver.rdoproject.org/99/54199/12/check/periodic-adoption-multinode-to-crc-ceph/78de32a/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_ceph_migration_out_2024-10-29T14:05:06EDT.log
		that failure looks unrelated to the external network changes under test (does it?)
		2024-10-29 18:05:06.556749 | TASK [Run data-plane-adoption Ceph migration tests from /home/zuul/src/github.com/openstack-k8s-operators/data-plane-adoption]
			TASK [ceph_migrate : ansible.builtin.fail if health is HEALTH_WARN || HEALTH_ERR] ***
			fatal: [np0004972941.localdomain]: FAILED! => {"changed": false, "msg": "Ceph is in HEALTH_WARN state."}
	- adoption tests passed
		so, we can stop doing the external network replacments for ctlplane in tripleo ceph roles as in the patch under test?
	https://logserver.rdoproject.org/99/54199/12/check/periodic-adoption-multinode-to-crc-ceph/78de32a/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_with_ceph_out_2024-10-29T12:47:12EDT.log
* periodic-adoption-multinode-to-crc-no-ceph-rollback: # should work when tripleo_network{} omitted, and tripleo netconfig skipped
	- overcloud deploy failed, the external network must be attached for tripleo in CI
	- ```

Reply:
The previous execution was OKish for the ceph job, however periodic-adoption-multinode-to-crc-no-ceph and the rollback jobs failed. It seems that attach external network flag didn't do the needful, so I adjusted the patch

Reply:
> we can safely reapply tripleo netconfig in CI idempotently

apparently, NOT. So I keep it off be default (retains the CI status quo)